sycl/include/CL/sycl/accessor.hpp

```diff
@@ -851,7 +851,7 @@ class accessor :
   void __init(ConcreteASPtrType Ptr, range<AdjustedDim> AccessRange,
               range<AdjustedDim> MemRange, id<AdjustedDim> Offset) {
     MData = Ptr;
-    for (int I = 0; I < AdjustedDim; ++I) {
+    [[clang::loop_unroll]] for (int I = 0; I < AdjustedDim; ++I) {
```

Comment:
Why did you decide to use clang attribute?
`#pragma` seems to be more portable.

Reply:
To make a claim, that we have done something useful, when implemented it :)

Reply:
I don't think, that `pragma` spelling is more portable: for example GCC has `pragma GCC unroll`, not `pragma unroll`.

Reply:
Isn't test added with implementation enough "to make a claim"?